scripts/scil_remove_invalid_streamlines.py

```diff
@@ -40,6 +40,9 @@ def _build_arg_parser():
                    help='Consider single point streamlines invalid.')
     p.add_argument('--remove_overlapping_points', action='store_true',
                    help='Consider streamlines with overlapping points invalid.')
+    p.add_argument('--threshold', type=float, default=0.001,
```

Comment:
@frheault does this default threshold makes sense to you ? 
Should we add an option to just remove overlapping points ?

Reply:
Since our smallest ever use step-size is 0.1, I could accept 0.01 (1/10 the smallest step-size we use).

But since almost all instance of overlapping point are perfect zero due to algorithmic error, this should catch everything anyway.